build.psm1

```diff
@@ -704,3 +714,12 @@ function Copy-CrossCompatibilityModule
         }
     }
 }
+
+Function RuleNameCompleter {
+    param($commandName, $parameterName, $wordToComplete, $commandAst, $fakeBoundParmeter)
+
+    Get-Childitem -Path "${projectRoot}\Tests\Rules" -Filter "*$wordToComplete*.tests.ps1" -Name |
```

Comment:
Might be here where the compatibility issue is occurring. And does the `-Name` parameter need an argument?

Reply:
This whole change should not have ended up here. I accidentally pushed a few files that did not belong. Reverted, and all good now.